### PR TITLE
Two update for get_deps_dir in date_dir

### PR DESCRIPTION
**Pull request by ypu**
Current get_deps_dir() can not work under python 2.4 env as python have some bug in that version. So make a work around for it. And also update the function for checking deps dirs as deps is under shared in virt-test, the original check will never find it.

### CODE REVIEW

**Comment by ypu** [comment]
ID: 1207544

File: virttest/data_dir.py [hunk 1]
```diff
@@ -182,13 +182,21 @@ def get_deps_dir(target=None):
...
     frame = inspect.stack()[1]
     # This is the module that called the function
-    module = inspect.getmodule(frame[0])
+    try:
+        module = inspect.getmodule(frame[0])
+        path = os.path.dirname(module.__file__)
+    except TypeError:
+        path = os.path.dirname(frame[1])
     # With the module path, we can keep searching with a parent dir with 'deps'
     # in it, which should be the correct deps directory.
```
Comment (PandaWei) [inline virttest/data_dir.py]: I think it's better to move the comments before module = inspect.getmodule(frame[0]) .
Others looks Good.
ACK.

**Comment by XueqiangWei** [comment]
LGTM,
Acked by Xueqiang Wei <xuwei@redhat.com>

**Comment by xiagao** [comment]
Acked-by: Xiaoling Gao <xiagao@redhat.com>

**Comment by ypu** [comment]
Thanks @PandaWei move the comments to above

**Comment by ypu** [comment]
Thanks @PandaWei @xiaoqing-wei @xiagao for your reviewing. As the second update only update some comments. So will mere it.

**Pull request merged**